pkgs/by-name/aw/awatcher/package.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+, gnomeSupport ? true
+, kwinSupport ? true
+}:
+
+rustPlatform.buildRustPackage rec {
+    pname = "awatcher";
+    version = "0.2.4";
+
+    src = fetchFromGitHub {
+    owner = "2e3s";
+    repo = pname;
```

```suggestion
    repo = "awatcher";
```
https://github.com/NixOS/nixpkgs/issues/277994